src/ResizeSensor.js

```diff
@@ -98,11 +98,15 @@
          * @returns {String|Number}
          */
         function getComputedStyle(element, prop) {
+            var computedElementStyle;
+            
             if (element.currentStyle) {
                 return element.currentStyle[prop];
             }
-            if (window.getComputedStyle) {
-                return window.getComputedStyle(element, null).getPropertyValue(prop);
+            
+            computedElementStyle = window.getComputedStyle(element, null);
+            if (window.getComputedStyle && computedElementStyle) {
```

Comment:
You access `window.getComputedStyle` in 107 which would fail if not existent. So the check in line 108 whether `window.getComputedStyle`exists makes no sense, since it either exists when this line is reached or it will never reach the line since it would error in line 107 before.